### DISCUSSION TITLE
Подвижное кресло.

### DISCUSSION THREAD

**Discussion by mitiaborodada** [vsulako/AFFBWheel#29] [General]
Приветствую!
Кто-нибудь задумывался над эффектом тряски для водительского сидения? 
Как вообще его можно реализовать? Я так понимаю, должна быть какая-то телеметрия от игры, на основании которой, контроллер управляет двигателями (гидравликами) кресла?

**Comment by lazenyuk-dmitry** [comment]
Привет. Я видел что используют моторы от дворников авто, штуки 3-4, для подвижной платформы. 

https://vk.me/join/AJQ1d13N7RYJuQvI4p/0fJGd
Вот тут как то обсуждали вроде. Можешь присоединиться, обсуждают вопросы там по самодельным рулям и тд.

Прошивку я думаю можно найти да и конструкцию если поискать. 

https://youtu.be/oqVrYiQ0cz4

**Comment by vsulako** [comment]
Вот такая штука есть https://opensfx.com/ для самосбора, на сервомоторах.
Управление через специальный плагин к игре, список поддерживаемых https://github.com/SimFeedback/SimFeedback-AC-Servo/wiki/Supported-Games
